### ISSUE TITLE
ResourcePool as measurement/dimension in influxdb

### ISSUE THREAD

**Issue by jakape** [Oxalide/vsphere-influxdb-go#12]
Hi there,

We would like to use the data in influxdb for customer reports. In order for this to work we need the ResourcePool metrics to be easily identifiable within influx/grafana. From your current example configuration, we thought this would be as easy as adding the following:
```json
                 {      
                    "ObjectType": [ "ResourcePool" ],
                        "Definition": [
                                { "Metric": "cpu.usagemhz.average", "Instances": "*" },
                                { "Metric": "mem.consumed.average", "Instances": "*" }
                        ]
                },
```

unfortunately, when we run the compiled binary. we see that this is picked up but it does seem that it is not reflected within influxdb.

```bash
 ~# vsphere-influxdb-go | grep -i resource
 (types.ManagedObjectReference) ClusterComputeResource:domain-c144,
 (types.ManagedObjectReference) ClusterComputeResource:domain-c7,
 (types.ManagedObjectReference) ClusterComputeResource:domain-c885,
 (types.ManagedObjectReference) ResourcePool:resgroup-886,
 (types.ManagedObjectReference) ResourcePool:resgroup-31635,
 (types.ManagedObjectReference) ResourcePool:resgroup-27811,
 (types.ManagedObjectReference) ResourcePool:resgroup-27806,
 (types.ManagedObjectReference) ResourcePool:resgroup-27800,
 (types.ManagedObjectReference) ResourcePool:resgroup-27814,
 (types.ManagedObjectReference) ResourcePool:resgroup-27813,
 (types.ManagedObjectReference) ResourcePool:resgroup-27805,
 (types.ManagedObjectReference) ResourcePool:resgroup-27810,
 (types.ManagedObjectReference) ResourcePool:resgroup-27812,
 (types.ManagedObjectReference) ResourcePool:resgroup-36683,
 (types.ManagedObjectReference) ResourcePool:resgroup-27802,
 (types.ManagedObjectReference) ResourcePool:resgroup-27803,
 (types.ManagedObjectReference) ResourcePool:resgroup-27808,
 (types.ManagedObjectReference) ResourcePool:resgroup-27815,
 (types.ManagedObjectReference) ResourcePool:resgroup-27809,
 (types.ManagedObjectReference) ResourcePool:resgroup-31697,
 (types.ManagedObjectReference) ResourcePool:resgroup-27807,
 (types.ManagedObjectReference) ResourcePool:resgroup-27804,
 (types.ManagedObjectReference) ResourcePool:resgroup-27801,
 (types.ManagedObjectReference) ResourcePool:resgroup-8,
 (types.ManagedObjectReference) ResourcePool:resgroup-145,

```

Within influx-db we cannot find these values in either measurements or series.

Do you have any idea on how we can manage to find the resource-pools ideally within the influxdb measurements?

Cheers,
J

**Comment by ampersize** [comment]
Looking into the same problem and trying to find a soultion - keep you posted.

**Comment by sofixa** [comment]
@jakape thanks for your input, @ampersize has created #16 and i'll merge as soon as the conflicts are resolved. Thanks, guys!

**Comment by sofixa** [comment]
Merged with #19
